roles/dcos_bootstrap/tasks/main.yml

```diff
@@ -124,13 +138,21 @@
 
 #  vvv DOWNLOAD vvv
 
-- name: Download installation file
+- name: Download Linux installation file
   get_url:
     url: "{{ dcos['download'] }}"
     checksum: "{{ dcos['download_checksum'] | default(omit) }}"
     dest: "{{ download_path }}/{{ install_file }}"
     mode: 0440
 
+- name: Download Windows installation file
```

Comment:
see my comment from above. I'm still a bit concerned about a second DC/OS download. This adds breakable complexity.

Reply:
please see my reply above.